tools/runners/test262-harness.py

```diff
@@ -927,6 +926,7 @@ def main():
 
 if __name__ == '__main__':
     try:
+        sys.stdout = io.TextIOWrapper(sys.stdout.buffer, encoding=sys.stdout.encoding, errors="ignore")
```

Comment:
Why does `stdout` need wrapping?

Reply:
because test262 cases have character can not represent in gbk and its print to console

Reply:
> Why does `stdout` need wrapping?

removed, fixed in another pr